nullaway/src/main/java/com/uber/nullaway/ErrorProneCLIFlagsConfig.java

```diff
@@ -200,6 +205,29 @@
               + FL_ACKNOWLEDGE_RESTRICTIVE
               + " is also set");
     }
+    fixSerializationActivationFlag = flags.getBoolean(FL_FIX_SERIALIZATION).orElse(false);
+    Optional<String> autoFixOutPutDir = flags.get(FL_FIX_SERIALIZATION_OUTPUT_DIR);
+    if (fixSerializationActivationFlag && !autoFixOutPutDir.isPresent()) {
+      throw new IllegalStateException(
+          "DO NOT report an issue to Error Prone for this crash!  NullAway Fix Serialization configuration is "
+              + "incorrect.  "
+              + "Must specify AutoFixer Output Directory, using the "
+              + "-XepOpt:"
+              + FL_FIX_SERIALIZATION_OUTPUT_DIR
+              + " flag.  If you feel you have gotten this message in error report an issue"
+              + " at https://github.com/uber/NullAway/issues.");
+    }
+    fixSerializationConfig =
```

Comment:
Shouldn't this only run if `fixSerializationActivationFlag` is true?

Reply:
If ```fixSerializationActivationFlag``` is ```false```, then the default constructor of  ```FixSerializationConfig``` is invoked and all features is set to ```false```. With this design, I avoided making ```fixSerializationConfig``` nullable. Please let me know if you believe I should only instantiate it if ```fixSerializationActivationFlag``` is ```true``` and leave it ```null``` otherwise.

Reply:
Are any methods of `fixSerializationConfig` invoked when `fixSerializationActivationFlag == false`?  I don't think they should be.  But in any case, I am ok with keeping `fixSerializationConfig` non-null, as long as we add a comment indicating what is going on.

Reply:
Yes no method is called when ```fixSerializationActivationFlag == false```. I added a comment and described the design intention. 79c1f6c16f9ee074a38796728a7c05f4a5ab7e04